libs/providers/flagd-web/project.json

```diff
@@ -76,7 +73,7 @@
       "executor": "nx:run-commands",
       "options": {
         "commands": [
-          "npx jest"
+          "npx jest --forceExit"
```

Comment:
some how jest did not exist with testcontainers

Reply:
`--forceExit` is bad, because it sometimes hides real bugs (it has before caught bugs with gRPC connections not dying, which can cause serious problems with reconnection in the provider). If we add `--forceExit`, we will not see such bugs.

Reply:
it seems like test containers or something did not end properly, i am not sure how to best debug this, we can investigate together if you like. the tests have been successful, but somehow a handle was still active, not sure which one ;)